modules/home/views/cta-section.tsx

```
@@ -0,0 +1,50 @@
+      
+      <div className="container mx-auto px-4 text-center relative z-10">
+        <div className="max-w-4xl mx-auto">
+          <h2 className="text-5xl md:text-7xl font-black text-foreground mb-6 leading-tight">
+            Take Control of Your
+            <span className="block text-primary">
+              Financial Future
+            </span>
+          </h2>
+          
+          <p className="text-xl text-muted-foreground mb-12 max-w-2xl mx-auto">
+            Join thousands of users who've transformed their financial habits with AI-powered expense tracking. Start your journey to financial freedom today.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)